utils/custom.js

```diff
@@ -123,16 +133,19 @@ const handleAnonymousData = async (data, req, postAuthorId, myAnonymousId, anony
       }
       return {
         ...child,
+        karmaScores: roundingKarmaScore(user?.karma_score || 0),
         is_you: req.userId === child.user_id,
         is_author: child.user_id === postAuthorId
       };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, there should be a type check for `user.karma_score` before passing it to `roundingKarmaScore`.

```diff
- karmaScores: roundingKarmaScore(user?.karma_score || 0),
+ karmaScores: typeof user?.karma_score === 'number' ? roundingKarmaScore(user.karma_score) : 0,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
already handle in the function